admin/services/pin.service.js

```diff
@@ -33,4 +36,17 @@ pinService.getActiveSchool = async (schoolId) => {
   return school
 }
 
+/**
+ * Expire pupil's pin
+ * @param token
+ * @param checkCode
+ */
+pinService.expirePupilPin = async (token, checkCode) => {
+  const decoded = jwtService.decode(token)
+  const pupil = await pupilDataService.findOne({_id: ObjectId(decoded.sub)})
+  const currentTimeStamp = moment.utc()
```

Comment:
does this work OK in unit tests?  would it not be better to use a date service (which i think we have) so you can mock this?

Reply:
@GuyHarwood Generally I noticed it works fine even if I set the date/time through sinon faketimers.
Besides I only need to verify that the pupil update call is fired for my current tests in case the user account is non test one.
Also so far we currently only use the date service for formatting purposes.

Reply:
If moment is working correctly with sinon then that’s ok 👍